.eslintrc.yml

```diff
@@ -45,3 +45,11 @@ rules:
   # these avoid React is defined but not used error, and is set in plugin:react/recommended
   # react/jsx-uses-react: 2
   # react/react-in-jsx-scope: 2
+
+  react/jsx-filename-extension:
+    - 1
```

Comment:
airbnb's rule only allows jsx syntax inside .jsx extension files

https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb/rules/react.js#L250

so we need to add this overwrite, but with `error`, 1 means warning,
using number is always confusing, so I would suggest to use semantic setting here.

Reply:
rest rules here are old setting, I would fix them after this PR